File: hot-framework/src/test/resources/be/icode/hot/test/nio/http/js/https-client-server-pkcs12.js
var options = {
	url: 'https://localhost:8443/test',
	ca:'ca/ca.crt',
	p12:"ca/client.p12",
	passphrase:"client"
}

hprint("Starting...")

client.buildRequest(options).done(function(data, status, response) {
	hprint(data)
	server.stop()
})

File: hot-framework/src/test/resources/be/solidx/hot/test/nio/python/https-client-server-pkcs12.py
options = {
	'hostname': 'localhost',
	'port': 8443,
	'path': "/test",
	'ca':'ca/ca.crt',
	'p12':"ca/client.p12",
	'passphrase':"client",
}

class Body:
	pass
	
def responseHandler(response):

    print "connected"
    body = Body()
    body.content = ""
	
    def dataHandler (chunck):
        body.content += chunck

    def endHandler ():
        print body.content
        print response.statusCode

    response.on("data",dataHandler)
    response.on("end", endHandler)

request = client.request (options , responseHandler)
request.end()